tests/network/vmi_networking.go

```diff
@@ -719,8 +719,8 @@ var _ = SIGDescribe("[Serial][rfe_id:694][crit:medium][vendor:cnv-qe@redhat.com]
 				Expect(libnet.PingFromVMConsole(serverVMI, ipAddr)).To(Succeed())
 
 				By("Checking ping (IPv4) to google")
-				Expect(libnet.PingFromVMConsole(serverVMI, "8.8.8.8")).To(Succeed())
-				Expect(libnet.PingFromVMConsole(clientVMI, "google.com")).To(Succeed())
+				Expect(libnet.PingFromVMConsole(serverVMI, "8.8.8.8", "-c 5", "-w 20")).To(Succeed())
```

Comment:
Should the `RunCommand` timeout be increased as well?

Reply:
updated here
https://github.com/kubevirt/kubevirt/pull/5430/files#diff-33a109786822bc5d06c2ce5108e77e7c84d7baa0137fcfa95e9deea7c3f2f438R38

If thats what u mean ?

Reply:
In this PR I would change the 8.8.8.8 -w to be less than the max command timeout. The dynamic calculation of the max command timeout can be done in a separate PR.

So just  to make sure I understand. `ping` with `-c` and `-w` will wait for `-c` value of echo replies till `-w` timeout expires? It means that more than `-c` value echo requests may be sent?

Reply:
> In this PR I would change the 8.8.8.8 -w to be less than the max command timeout. The dynamic calculation of the max command timeout can be done in a separate PR.

Done thanks
 
> So just to make sure I understand. `ping` with `-c` and `-w` will wait for `-c` value of echo replies till `-w` timeout expires? It means that more than `-c` value echo requests may be sent?

yes, and yes
example

```
$ ping machine -c 1 -w 200
PING machine (...) 56(84) bytes of data.
64 bytes from machine (...): icmp_seq=173 ttl=57 time=18.0 ms

--- ... ping statistics ---
173 packets transmitted, 1 received, 99% packet loss, time 172002ms
rtt min/avg/max/mdev = 18.015/18.015/18.015/0.000 ms
$ echo $?
0
```

Server was down
Started ping
Started server
waiting for only 1 packet, but actually 173 sent, once 1 got, it finished successfully.